moped-editor/src/views/staff/StaffListViewFiltersConf.js

```diff
@@ -63,15 +63,6 @@ export const StaffListViewFiltersConf = {
         "*", // All of them (shortcut)
       ],
     },
-    {
-      name: "is_coa_staff",
-      label: "Is COA Staff",
-      placeholder: "Select Staff",
-      type: "boolean",
-      operators: [
-        "*", // All of them (shortcut)
-      ],
-    },
```

Comment:
this filter was not working and since everyone initially will be COA Staff, the filter is unnecessary